moveit_ros/moveit_servo/include/moveit_servo/servo_calcs.h

```diff
@@ -154,9 +154,6 @@ class ServoCalcs
   void suddenHalt(trajectory_msgs::msg::JointTrajectory& joint_trajectory) const;
   void suddenHalt(sensor_msgs::msg::JointState& joint_state) const;
 
-  /** \brief  Scale the delta theta to match joint velocity/acceleration limits */
-  void enforceVelLimits(Eigen::ArrayXd& delta_theta);
-
   /** \brief Avoid overshooting joint limits */
   bool enforcePositionLimits(sensor_msgs::msg::JointState& joint_state) const;
```

Comment:
Seems like this should be moved to the new `enforce_limits.cpp too`

Reply:
I agree.  This was more about converting the test to a unit test instead of a ros test (as those take forever and are flaky on foxy).

Long term I think it would be nice to separate all the math/logic from the ros code so it could be easily tested.